classes/install/PKPInstall.inc.php

```diff
@@ -299,10 +300,10 @@ function createData() {
 		}
 
 		// Install default site settings
-		$siteSettingsDao = DAORegistry::getDAO('SiteSettingsDAO');
-		$siteSettingsDao->installSettings('registry/siteSettings.xml', array(
-			'contactEmail' => $this->getParam('adminEmail')
-		));
+		$schemaService = ServicesContainer::instance()->get('schema');
+		$site = $schemaService->setDefaults(SCHEMA_SITE, $site, $site->getSupportedLocales(), $site->getPrimaryLocale());
+		$site->setData('contactEmail', $this->getParam('adminEmail'));
+		$siteDao->updateobject($site);
```

Comment:
`updateObject` (capitalization, not that it matters much)

Reply:
:+1: